servers/src/main/java/tachyon/UserInfo.java

```diff
@@ -54,4 +54,17 @@ public synchronized String toString() {
     sb.append(")");
     return sb.toString();
   }
+  
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
```

Comment:
Quote from Google java style (which is adopted by Tachyon with slight modification)
>Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.